src/components/Modals/CustomLayoutModal/CustomLayoutModal.tsx

```diff
@@ -27,6 +27,7 @@ export default class CustomLayoutModal extends React.PureComponent<Props, State>
   handleBack = () => {
     const { step } = this.state
     switch (step) {
+      // TODO: remove this after removing the SDK7_TEMPLATES feature flag
```

Comment:
I left several of these comments through the codebase to remember that once we remove the `SDK7_TEMPLATES` (because we won't want SDK6 templates anymore) then these are parts of the code that don't need to exist anymore (like the SDK version selection step)